client/http/interface.go

```diff
@@ -337,6 +338,20 @@ func (c *client) SetStoreLabels(ctx context.Context, storeID int64, storeLabels
 		WithBody(jsonInput))
 }
 
+// GetHealthStatus gets the health status of the cluster.
+func (c *client) GetHealthStatus(ctx context.Context) ([]Health, error) {
```

Comment:
BTW, could you please add tests for this new method in `/tests/integrations/client/http_client_test.go`?

Reply:
https://github.com/tikv/pd/pull/8212/commits/b1234949e72339bca5805ea7533d7caefd59d159